src/widget/weffectselector.cpp

```diff
@@ -72,7 +72,7 @@ void WEffectSelector::populate() {
     }
 
     //: Displayed when no effect is loaded
-    addItem(tr("None"), QVariant());
+    addItem(MIXXX_NOEFFECT_STRING, QVariant());
```

Comment:
Throughout Mixxx different strings are used as placeholders: `QVariant(QString(""))`, `QString("")` or `QVariant()`
What's safe/correct?

Reply:
https://doc.qt.io/qt-5/qcombobox.html#addItem

I think you can just omit the last parameter and it defaults to QVariant() if the user data is not uses. 
userData is just a container that can store any extra data along with the string.